static/app/views/dashboardsV2/dashboard.spec.tsx

```diff
@@ -290,21 +293,29 @@ describe('Dashboards > Dashboard', () => {
       widgets = [newWidget];
     });
 
-    it('displays the copy widget button in edit mode', () => {
+    it('displays the copy widget button in edit mode', async () => {
       const dashboardWithOneWidget = {...mockDashboard, widgets};
-      mount(dashboardWithOneWidget);
+      await act(async () => {
+        mount(dashboardWithOneWidget);
+        await tick();
+      });
```

Comment:
heads up @getsentry/discover-n-dashboards if you see `act` errors, this is how I fixed them